CONTRIBUTING.md

```diff
@@ -43,6 +43,12 @@ You can set up and run these tools locally (see [Travis script] for the details)
 Generally, we follow common best practices established in the Rust community,
 but we have several additional conventions:
 
+- Create as minimal pull request as possible: they are easier to review and
+  integrate. 
+
+  Additionally, we merge pull request using the "squash and merge" strategy, so
+  don't waste your time on detailed commits history.
```

Comment:
But sound commit history make it easier to review PRs (at least from my experience). On a related note: should we suggest to use merges from `master` instead of rebasing since we're using "squash and merge" anyway?

Finally, this is at odds with the contributing page on the Exonum website, which spends much time describing how commits should be named, etc. Would you mind updating it?

Reply:
> But sound commit history make it easier to review PRs

yep, but the idea is to have small PRs in the first place

Reply:
@slowli 
> But sound commit history make it easier to review PRs (at least from my experience).

Makes sense, though I rarely check pull requests commit by commit (I prefer to see the whole scope of changes).

> Would you mind updating it?

Sure, I'm going to send a pull request soon.